specification/adp/resource-manager/readme.python.md

```diff
@@ -26,10 +27,10 @@ clear-output-folder: false
 perform-load: false
 ```
 
-``` yaml $(tag) == 'package-2020-07-01-preview'
-namespace: azure.mgmt.adp.v2020_07_01_preview
-output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp/azure/mgmt/adp/v2020_07_01_preview
+``` yaml
```

Comment:
we need to keep this explicit to make things decoupled, therefore I suggest we duplicate this snippet each for one tag
```suggestion
``` yaml $(tag) == 'package-2020-07-01-preview'
```